relayer/chains/substrate/substrate_chain_processor.go

```diff
@@ -184,19 +186,49 @@ func (scp *SubstrateChainProcessor) Run(ctx context.Context, initialBlockHistory
 		return err
 	}
 
-	scp.log.Debug("Entering main query loop")
+	scp.log.Debug("subscribing and listening to relay chain commitments")
+	commitments := make(chan interface{})
+	sub, err := scp.chainProvider.RelayChainRPCClient.Client.Subscribe(
+		context.Background(),
+		"beefy",
+		"subscribeJustifications",
+		"unsubscribeJustifications",
+		"justifications",
```

Comment:
I think making these constant values would be better long term vs. using string literals but not a blocker on merging rn

Reply:
I'll resolve them in PR #1052